lib/token.ts

```diff
@@ -673,17 +673,22 @@ function getWithRedirect(sdk: OktaAuth, options: TokenParams): Promise<void> {
       var urls = getOAuthUrls(sdk, options);
       var requestUrl = urls.authorizeUrl + buildAuthorizeParams(tokenParams);
 
-      // Set session cookie to store the oauthParams
+      // Store the oauthParams in storage for re-visiting when redirect back
```

Comment:
I _think_ this code would only be used in front end SDKs, not by the widget, so there's no concern with having duplicate and out-of-sync copies of auth-js, but I'd be happier if someone else explicitly agreed